.github/workflows/docker-verify.yaml

```
@@ -0,0 +1,28 @@
+name: Docker verify process
```

nit: let's make the name a bit more descriptive, maybe something like

```suggestion
name: Dockerfile linter
```

---

I would pospone this conversation until we finish with #690 . This GHA will contain more jobs apart from the lint.

---

Won't those be separate actions?

---

Mmm... Not sure, maybe not. We have different options:
- One action (docker-verify) with different jobs (lint, sast, vscan)
- Split them in actions (docker-lint, docker-sast, docker-vscan)

My idea was to start with the first option. Or there is even a third option: include the docker checks in each `{project}-verify`.

---

With #625 maybe, the third option would be my preferred one (maybe). But with our current structure I think the first option is the easiest one.